desktop/core/src/desktop/js/apps/newimporter/app.js

```diff
@@ -0,0 +1,27 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import huePubSub from '../../utils/huePubSub';
+
+huePubSub.subscribe('app.dom.loaded', app => {
+  if (app === 'newimporter') {
+    window.createReactComponents('#embeddable_newimporter');
+    return;
+  }
+  if (app !== 'importer') {
+    return;
+  }
+  window.createReactComponents('#embeddable_importer');
```

Comment:
Since you added the new importer under a new name I don't think you need to care about the old one here meaning these lines 23:26 could be removed, or?